resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -173,7 +173,7 @@ function task_install {
     ln -s "../apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${symbindir}/<%= basename %>"
     ln -s "../server/apps/${real_name}/bin/<%= basename %>" "${DESTDIR}${uxbindir}/<%= basename %>"
 <% end -%>
-    install -d -m 0755 "${DESTDIR}${rundir}"
+    install -d -m 0755 -o <%= EZBake::Config[:user] %> -g <%= EZBake::Config[:group] %> "${DESTDIR}${rundir}"
```

Comment:
Hmm, this appears to have caused a package build failure for both RedHat and Debian for me.  From a build of pe-puppet-server-extensions at http://jenkins-release.delivery.puppetlabs.net/job/pe-puppetdb-packaging-2016-10-12-21-48-52-4.3.0.SNAPSHOT.2016.10.12T2148 for a Debian platform, I saw:

~~~~
+++ install -d -m 0755 -o pe-puppetdb -g pe-puppetdb /tmp/buildd/pe-puppetdb-4.3.0/debian/pe-puppetdb/var/run/puppetlabs/puppetdb
install: invalid user 'pe-puppetdb'
make: *** [install/pe-puppetdb] Error 1
~~~~

I suppose this makes sense if we don't have fixed uids/gids for the user/group and those are determined instead when the useradd/groupadd command is run at package install time.

If we're resetting the permissions at service start time / tmpfiles.d, I'm wondering now if we'll even need this anyway.